nixos/modules/config/zram.nix

```diff
@@ -105,6 +114,22 @@ in
       }
     ];
 
+    # https://wiki.archlinux.org/title/Zram#Optimizing_swap_on_zram
+    boot.kernel.sysctl = lib.mkIf cfg.recommendedSysctlSettings {
+      # https://github.com/pop-os/default-settings/blob/041cd94158142d6a34d2e684c847ac239a5ba086/etc/sysctl.d/10-pop-default-settings.conf#L2-L3
+      # These values are what PopOS always uses, Perhaps they could be moved to `nixos/modules/config/sysctl.nix`
```

Comment:
I'd prefer an explanation of *why* these are used rather than a link to another distro.

Reply:
I think those values were based on this benchmark?
https://www.reddit.com/r/Fedora/comments/mzun99/new_zram_tuning_benchmarks/

Reply:
The benchmark is also linked below.